### PR TITLE
refactor(dev): add misc fixes that clean up test style 

### PR DESCRIPTION
**Pull request by kptkin**
Description
-----------
What does the PR do?

This PR just clean up some stylistic issues in the test and typos

Testing
-------
How was this PR tested?

This clean up tests so current testing is sufficient

### CODE REVIEW

File: tests/test_mode_disabled.py [hunk 1]
```diff
@@ -71,8 +72,8 @@ def test_disabled_ops(test_settings):
 
 def test_disabled_dir(test_settings, mocker):
```
Comment (dmitryduev) [inline tests/test_mode_disabled.py:73]: Do we need `mocker` at all?

File: tests/test_save_policies.py [hunk 2]
```diff
@@ -31,7 +30,7 @@ def spoof_min_wait_for_size(livePolicy, size):
     yield livePolicy
 
 
-def test_policy_on_modified(monkeypatch, wandb_init_run, mocked_live_policy):
```
Comment (dmitryduev) [inline tests/test_save_policies.py:34]: very happy to see it go :)

File: tests/wandb_integration_test.py [hunk 3]
```diff
@@ -444,27 +444,23 @@ def mock_min_size(self, size):
...
         fp.write("a" * 10000)
-        fp.close()
     # 2.1 seconds is longer than set rate limit
     sent += os.path.getsize(file_path)
     # give watchdog time to register the change
     time.sleep(1.0)
     # file updated within modified time, should not be uploaded
     with open(file_path, "a") as fp:
         fp.write("a" * 10000)
-        fp.close()
```
Comment (dmitryduev) [inline tests/wandb_integration_test.py:462]: huh ;)

**Review by dmitryduev — APPROVED** [review]
Thank you @kptkin!!

File: tests/test_mode_disabled.py (hunk 1 above)
Comment (kptkin) [reply tests/test_mode_disabled.py:73]: ah yeah we don't... i'll remove

**Pull request merged**